src/unittest/test_serialization.cpp

```diff
@@ -708,6 +711,51 @@ void TestSerialization::testFloatFormat()
 		UASSERT(test_single(i));
 }
 
+void TestSerialization::testDoubleFormat()
+{
+	std::ostringstream os(std::ios_base::binary);
+
+	for (int i = 0; i < 150; i++)
+	{
+		writeD1000(os, test_double_data[i]);
+		writeD(os, test_double_data[i]);
+		writeF32(os, test_double_data[i]);
+		writeF32(os, test_double_data[i]);
+	}
+	for (int i = 0; i < 150; i += 3)
+	{
+		v3d p(test_double_data[i], test_double_data[i+1], test_double_data[i+2]);
+		writeV3D1000(os, p);
+		writeV3D(os, p);
```

Comment:
I don't see much point in testing writeV3D more than once, if writeD works it will also work.

Reply:
Fair enough. I will remove the extra tests.